micro_sam/training/util.py

```diff
@@ -258,19 +261,20 @@ def __init__(self, desired_shape, do_rescaling=False, padding="constant"):
         self.do_rescaling = do_rescaling
 
     def __call__(self, raw):
+        raw = to_rgb(raw)  # Ensure all images are in 3-channels: triplicate one channel to three channels.
+
         if self.do_rescaling:
-            raw = normalize_percentile(raw, axis=(1, 2))
-            raw = np.mean(raw, axis=0)
+            # NOTE: Below is done for TissueNet: to work with the valid channels (i.e. the first and second channels).
```

Comment:
This is a bit hacky. Can we properly expose this? E.g. add a parameter `valid_channels`?